Makefile.am

```diff
@@ -128,6 +130,38 @@ crypt-hashes.h: hashes.lst gen-hashes.awk Makefile
 	  $(srcdir)/hashes.lst > crypt-hashes.h.T
 	$(AM_V_at)mv -f crypt-hashes.h.T crypt-hashes.h
 
+install_exec_hook_targets =
+
+if ENABLE_STATIC
```

Comment:
I bet the ALT people are going to want to be able to turn this off independently :)